internal/functions/efgs/keys-operations.go

```diff
@@ -96,7 +91,7 @@ func signBatch(ctx context.Context, diagnosisKey *DiagnosisKeyBatch) (string, er
 
 	nbbsPair, err := efgsutils.LoadX509KeyPair(ctx, efgsutils.NBBS)
 	if err != nil {
-		logger.Errorf("Error loading authentication certificate: %v", err)
+		logger.Debugf("Error loading authentication certificate.")
```

Comment:
Please do include the error here in the message. Keep the previous version, just change error to debug.